### DISCUSSION TITLE
请问后续会支撑国产数据库吗？

### DISCUSSION THREAD

**Discussion by wangguancong** [hhyo/Archery#2576] [General]
请问后续会支撑国产数据库吗？

**Comment by LeoQuote** [comment]
你好, 本项目是一个开源项目, 你可以自由提功能需求, 但其实比提功能需求更加务实, 有价值的是实现这个功能, 本项目开发文档如下, 希望对你有帮助:

* https://github.com/hhyo/Archery/wiki/development
* https://github.com/hhyo/Archery/wiki/notifier

期待你的pr, 谢谢~
